plasmapy/physics/transport.py

```diff
@@ -236,6 +238,37 @@ class classical_transport:
     by these other effects. Thus, classical transport often serves as a lower
     bound on the losses / transport encountered in a plasma.
 
+    Models implemented:
+
+    * Braginskii [1]
+    The original Braginskii treatment as presented in the highly cited review
+    paper from 1965. Coefficients are found from expansion of the kinetic
+    equation in Laguerre / Sonine polynomials, truncated at K = 2. This theory
+    allow for arbitrary Hall parameter and include results for Z = 1, 2, 3, 4,
+    and infinity (Lorentz gas / stationary ion approximation.)
+
+    * Spitzer-Harm [2,3]
+    These coefficients were obtained from a numerical solution of the Fokker-
+    Planck equation. They give one of the earliest and most accurate (in the
+    Fokker-Planck sense) results for electron transport in simple plasma. They
+    principally apply in the unmagnetized / parallel field case, although for
+    resistivity Spitzer also calculated a famous result for a strong
+    perpendicular magnetic field. Results are for Z = 1, 2, 4, 16,
```

Comment:
I just want to say that this is really neatly written. I'd read more of it! :D

Reply:
Thanks! :)